### PR TITLE
Refactor twitter

### PR DESCRIPTION
**Pull request by chenzongxiong**
1. Send a direct message to user's Twitter 
2. add a periodic task to prevent unexcepted errors occur.

### CODE REVIEW

**Review by geastwood — APPROVED** [review]
LGTM, good work.

File: app/validator/twitter.js [hunk 1]
```diff
@@ -1,91 +1,124 @@
...
+        try {
+            const resp = await this.sendCtaMessage(twitterAccount, link);
+            logger.debug(`Send verification message to ${JSON.stringify(resp)} successfully.`);
+            return resp;
+        } catch (error) {
+            const errorData = JSON.parse(error.data);
+            /* eslint-disable-next-line */
+            if (errorData.hasOwnProperty('errors')) {
+                for (let e of errorData.errors) {
+                    if (e.code === 17) {
```
Comment (h4n0) [inline app/validator/twitter.js:39]: If I understand it correctly, by default we cannot send DM to any user that hasn't followed us. So this is a very common case I guess.

**Review by h4n0 — COMMENTED** [review]
Please see my inline comment.

File: app/validator/twitter.js (hunk 1 above)
Comment (chenzongxiong) [reply app/validator/twitter.js:39]: Yes. We should let the user follow us first. 
Or the user allows receiving any private conversation in his privacy settings. 
The biggest problem is that the user turns off this setting by default. So it backs to the original problem, we have to let the user search our official Twitter account.

**Pull request merged**